fixmydjango/fixmydjango
# Refs #49 - "Preview" button in the comment editor shows login page

Link to task: https://github.com/fixmydjango/fixmydjango/issues/49

How to test:
- Make sure to be logged in
- Enter in any error page
- Try to preview a comment

Preview box:
![preview](https://cloud.githubusercontent.com/assets/8882260/23262186/9897ec98-f9b8-11e6-87ae-f6923a25d9b6.png)
